app/views/content/funding-and-support/scholarships-and-bursaries.md

```diff
@@ -85,46 +69,56 @@ To be eligible for a scholarship, you’ll need at least one of the following:
 * a master’s degree
 * a PhD
 
-In exceptional circumstances, if you achieved a 2:2 in your degree you may still be awarded a scholarship if you have significant relevant experience.
+If you achieved a 2:2 in your degree you may get a scholarship if you have enough relevant experience.
```

Comment:
```suggestion
If you achieved a 2:2 in your degree you may get a scholarship if you have enough relevant experience.
```
not sure if `get` is the right word here, given that you have to apply for it? we currently say `be awarded`